File: JavaScript/M01_JavaScriptFundamentals/L04_Functions/Lab/P02_Grades.js
function grades(num) {
    if (num >= 2.00 && num <= 2.99) {
        console.log(`Fail`);
    } else if (num >= 3.00 && num <= 3.49) {
        console.log(`Poor`);
    } else if (num >= 3.50 && num <= 4.49) {
        console.log(`Good`);
    } else if (num >= 4.50 && num <= 5.49) {
        console.log(`Very good`);
    } else if (num >= 5.50 && num <= 6.00) {
        console.log(`Excellent`);
    }
}

File: C++/M02_Fundamentals/L02_Functions/Lab/Solutions/P02_Grades.cpp
#include <iostream>
#include <string>

std::string getGrade(double grade) {

	std::string output = "";

	if (2 <= grade && grade <= 2.99) {
		output = "Fail";
	} else if (3 <= grade && grade <= 3.49) {
		output = "Poor";
	} else if (3.5 <= grade && grade <= 4.49) {
		output = "Good";
	} else if (4.5 <= grade && grade <= 5.49) {
		output = "Very good";
	} else if (5.5 <= grade && grade <= 6) {
		output = "Excellent";
	}

	return output;
}


int main() {

	double num;
	std::cin >> num;

	std::cout << getGrade(num) << std::endl;

	return 0;
}